treestats_paper.tex

```diff
@@ -856,23 +856,24 @@ \section*{Duality of site and branch statistics}
 use the same summary function $f(x) = x(n-x)/\left(n(n-1)\right)$.
 These are closely related because under an infinite-sites model of mutation,
 two sequences differ at a site only if there has been a mutation somewhere on the branches going back
-to their most recent common ancestor,
-and so if mutations occur with constant rate,
-the expected value of genetic diversity,
-averaging over mutational noise given the tree sequence,
-is equal to the mutation rate multiplied by the average distance between the two in the trees.
+to their most recent common ancestor.
+Therefore, if mutations occur with constant rate
+the expected value of genetic diversity
+% averaging over mutational noise given the tree sequence,
+is equal to the mutation rate multiplied by the average path distance between
+the two samples in the trees.
 
 This relationship is true more generally.
 In fact, for any region of the genome between $i$ and $j$,
-with $\treeseq_{[i,j)}$ the trees spanning that interval,
+% with $\treeseq_{[i,j)}$ the trees spanning that interval,
```

Comment:
hm, but we need to define this notation somewhere?

Reply:
maybe we define it in a separate short sentence after the equation?

Reply:
will merge and then deal with this

Reply:
I figured we had defined it earlier on, but maybe we didn't. I'm easy either way.

Reply:
I fixed this up in #50.